index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="refresh" content="0;url=demo/" />
+    <title>demo-element</title>
```

Comment:
I think the title `icon-toggle` is fine here—this file is supposed to describe the project.

It doesn't really matter much, though.